client/src/sampleData.js

```diff
@@ -0,0 +1,153 @@
+const sampleData = {
+  products: [
```

Comment:
Can we add sample data of the product information and product styles as well?

Reply:
Done